Nosach/web-quiz-project/src/main/java/com/nosach/quizproject/service/QuizService.java

```diff
@@ -0,0 +1,39 @@
+package com.nosach.quizproject.service;
+
+import com.nosach.quizproject.dao.QuizDAO;
+import com.nosach.quizproject.entity.Quiz;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+@Component
+@Transactional
```

Comment:
Better to mark transactional methods as transactional, not whole class

Reply:
Ok